Social Distancing Measures.ipynb

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
<p>Do you think it'd be better to model this in two splits? One relaxed lockdown from April 20 and completely relaxed from May 2?</p><p> </p><p>How do we model these two scenarios? What is Day 0 and Day 25 in real dates?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/coronaIndia/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Q1: Sure, I can incorporate this by relaxing the scale parameter of my graph defined at the starting of the notebook. I can increase the scale from 10 to 20 for a relaxes lock-down type scenario</p><p> </p><p>Q2: The day is for April 18th at the moment. In the new commit, day 0 is April 18th, a lock-down of 15 days till May 3rd and then trend for 85 more days </p>

Reply:
How can we rely that these params work for us? Can we use Lombardo or Wuhan for reference and get corresponding Day 0 curves and see how they compare against reality?